.github/workflows/dependency-review.yml

```
@@ -0,0 +1,39 @@
+  dependency-review:
+    runs-on: ubuntu-latest
+    steps:
+      - name: 'Checkout repository'
+        uses: actions/checkout@v4
+      - name: 'Dependency Review'
+        uses: actions/dependency-review-action@v4
+        # Commonly enabled options, see https://github.com/actions/dependency-review-action#configuration-options for all available options.
+        with:
+          comment-summary-in-pr: always
+          fail-on-severity: low
+        #   deny-licenses: GPL-1.0-or-later, LGPL-2.0-or-later
```

YAMLlint (1.38.0)

[warning] comment not indented like content

(comments-indentation)